notebooks/vertex_genai/solutions/promp_engineering.ipynb

```diff
@@ -5,10 +5,19 @@
    "id": "02aa307c-a2b7-4927-98f5-51b27b908ba0",
```

Comment:
<p>Should we add a comment here mentioning which version of aiplatform it was tested and worked on?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/399/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done!</p>